services/QuillLMS/client/app/bundles/Comprehension/components/studentView/readAndHighlightInstructions.tsx

```diff
@@ -1,46 +1,61 @@
 import * as React from 'react'
 
-import { DEFAULT_HIGHLIGHT_PROMPT, } from '../../../Shared/utils/constants'
+import DisplayStudentHighlight from './displayStudentHighlight'
+import DirectionsSectionAndModal from './directionsSectionAndModal'
+
 import { cursorClick, cursorPointingHand, } from '../../../Shared/index'
 
-const ReadTheDirectionsModal = ({ closeReadTheDirectionsModal, }) => (<section className="read-the-directions-modal">
-  <p>Read the directions carefully before moving onto reading and highlighting.</p>
-  <button className="quill-button primary contained large focus-on-light" onClick={closeReadTheDirectionsModal} type="button">Got it</button>
-</section>)
+const HighlightInstructions = () => (
+  <div className="highlight-instructions">
+    <h4>How to highlight</h4>
+    <div className="highlight-instruction one">
+      <h5 className="highlight-instruction-header">
+        <span className="highlight-instruction-number">1</span>
+        <span>Hover over a sentence</span>
+      </h5>
+      <img alt={cursorPointingHand.src} src={cursorPointingHand.src} />
```

Comment:
should be `cursorPointingHand.alt` for the first one here

Reply:
Yup, will refactor for next PR.